src/eventHandlers.js

```diff
@@ -0,0 +1,73 @@
+import {
+  get
+} from 'lodash';
+
+import { coreEvents, updateUser } from '@folio/stripes/core';
+
+import AccessModal from './AccessModal';
+import ChangeServicePoint from './ChangeServicePoint';
+
+/**
+ * handleCheckServicePoints
+ * return true if servicePoints is a non-empty array
+ *
+ * @param {object} stripes
+ * @returns {booelan}
+ */
+export const handleCheckServicePoints = (stripes) => {
+  return get(stripes, ['okapi', 'loginData', 'servicePointsUser', 'servicePoints'], []).length > 0;
```

Comment:
It looks like you removed `get` in a couple of other places so maybe we can do it here too.

````js
(stripes?.okapi?.loginData ?.servicePointsUser?.servicePoints ?? []).length > 0
````